### DISCUSSION TITLE
Your application has been approved you will receive benefits soon coming from several days

### DISCUSSION THREAD

**Discussion by saadatgit1** [community/community#136919] [GitHub Education]
### Hi everyone,

I've applied for the student pack of github and the application has been approved but it is showing that you will receive benefits soon from several days. Please help me to get that benefits soon.

**Comment by Yackens** [comment]
You can only wait. I believe that it takes around 7 days to get all the benefits. Hope it helps 😄

**Comment by davevad93** [comment]
Hi @saadatgit1 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @saadatgit1 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
